aws/resource_aws_datapipeline_test.go

```diff
@@ -0,0 +1,269 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/datapipeline"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSDataPipeline_basic(t *testing.T) {
```

Comment:
To match the resource renaming:

```suggestion
func TestAccAWSDataPipelinePipeline_basic(t *testing.T) {
```